app/platform/u8x8_nodemcu_hal.c

```diff
@@ -9,6 +9,7 @@
 #include "c_stdlib.h"
 
 #include "platform.h"
+#include "cpu_esp8266.h"
```

Comment:
```suggestion
```
We get this by including `platform.h`, don't we?

Reply:
You're right, I didn't check that platform.h already has this include.